packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -46,6 +46,25 @@ export const asteroid = (model: ModelData): string[] => [
 model = BaseModel.from_pretrained("${model.id}")`,
 ];
 
+export const audioseal = (model: ModelData): string[] => [
+	`# Watermark Generator
+from audioseal import AudioSeal
+
+model = AudioSeal.load_generator("${model.id}")
+# pass a tensor (tensor_wav) of shape (batch, channels, samples) and a sample rate
+wav, sr = tensor_wav, 16000
+
+watermark = model.get_watermark(wav, sr)
+watermarked_audio = wav + watermark
+
+# Watermark Detector
```

Comment:
The function returns a list of strings, so you can actually return multiple snippets rather than hacking them in a single one (see `transformers` for example)

Reply:
Let me know if this doesn't work! 🤗